docs/source/_static/js/custom.js

```diff
@@ -40,158 +41,209 @@ function isValidFragment(splitFragment) {
+				.querySelector(".sidebar-drawer")
+				.classList.remove("hide-sidebar");
+		}
+	});
+	window.matchMedia("(max-width: 82em)").addEventListener("change", (event) => {
+		if (!event.matches) {
+			document.querySelector(".toc-drawer").classList.remove("hide-sidebar");
+		}
+	});
 }
 
 // Activates labels when a user focuses on them and clicks "Enter".
 // Also highlights the next appropriate input label.
 function activateLabelOnEnter() {
-  const labels = document.querySelectorAll('label');
-  labels.forEach((element) => {
-	element.setAttribute('tabindex', '0');
-    element.addEventListener('keypress', (event) => {
-      if (event.key === 'Enter') {
-        const targetId = element.getAttribute('for');
-        document.getElementById(targetId).click();
-        const nextFocusId = NEXT_FOCUS_ID_MAP[element.id];
-        if (nextFocusId) {
-          // Timeout is needed to let the label become visible.
-          setTimeout(() => {
-            document.getElementById(nextFocusId).focus();
-          }, 250);
-        }
-      }
-    });
-  });
+	const labels = document.querySelectorAll("label");
+	labels.forEach((element) => {
+		element.setAttribute("tabindex", "0");
+		element.addEventListener("keypress", (event) => {
+			if (event.key === "Enter") {
+				const targetId = element.getAttribute("for");
+				document.getElementById(targetId).click();
+				const nextFocusId = NEXT_FOCUS_ID_MAP[element.id];
+				if (nextFocusId) {
+					// Timeout is needed to let the label become visible.
+					setTimeout(() => {
+						document.getElementById(nextFocusId).focus();
+					}, 250);
+				}
+			}
+		});
+	});
+}
+
+function loadThemeFromLocalStorage() {
+	document.body.dataset.theme = localStorage.getItem("theme") || "auto";
 }
 
 // Improves accessibility for keyboard-only users.
 function setupKeyboardFriendlyNavigation() {
-  activateLabelOnEnter();
-  toggleSidebarMenuVisibility('.toc-drawer', '#__toc');
-  toggleSidebarMenuVisibility('.sidebar-drawer', '#__navigation');
+	activateLabelOnEnter();
+	toggleSidebarMenuVisibility(".toc-drawer", "#__toc");
+	toggleSidebarMenuVisibility(".sidebar-drawer", "#__navigation");
 }
+
+function loadShortbread() {
+	if (typeof AWSCShortbread !== "undefined") {
+		const shortbread = AWSCShortbread({
+			// If you're testing in your dev environment, use ".cloudfront.net" for domain, else ".amazonaws.com"
+			domain: ".amazonaws.com",
+		});
+
+		// Check for cookie consent
+		shortbread.checkForCookieConsent();
+
+		const cookiePreferencesLink = document.getElementById("cookie-button-link");
+		if (cookiePreferencesLink) {
+			cookiePreferencesLink.addEventListener("click", function (event) {
+				event.preventDefault();
+				shortbread.customizeCookies();
+			});
+		}
+
+		console.log("AWSCShortbread successfully loaded...!!!");
+	} else {
+		console.error("AWSCShortbread failed to load!!!");
+	}
+}
+
 // Functions to run after the DOM loads.
 function runAfterDOMLoads() {
 	expandSubMenu();
+
```

Comment:
nit - Just curious, why does this empty line exist? What this change added by you or the formatter?